app/views/invoices/new.html.haml

```diff
@@ -5,7 +5,7 @@
         = render 'shared/content_title', title: "Create a New Invoice" 
         = render 'shared/error_messages', obj: @invoice
         %fieldset
-          = f.hidden_field :invoice_num, value: Invoice.count + 50001
+          = f.hidden_field :invoice_num, value: Invoice.count + 50010
```

Comment:
The only change you might consider making to this PR is giving `50010` a name. From your PR title, maybe something like `Invoice::STARTING_POINT` or `Invoice::INITIAL_SEED` (to borrow from minter language). Just a thought.